packages/playwright/src/runner/reporters.ts

```diff
@@ -160,6 +160,6 @@ class ListModeReporter implements ReporterV2 {
 
   onError(error: TestError) {
     // eslint-disable-next-line no-console
-    console.error('\n' + formatError(error, false).message);
+    console.error('\n' + formatError(terminalScreen, error).message);
```

Comment:
It looks like we did not want colors here at all, which seems consistent with plain `console.log()` above instead of using `formatTitle()`. Should we have a separate `noColorsScreen`?